src/DepthCameraSensor.cc

```diff
@@ -299,15 +299,6 @@ bool DepthCameraSensor::Load(const sdf::Sensor &_sdf)
   gzdbg << "Points for [" << this->Name() << "] advertised on ["
          << this->Topic() << "/points]" << std::endl;
 
-  // Initialize the point message.
```

Comment:
The way this is handled in `CameraSensor.cc` is that `OpticalFrameId` is only used if it's specified by the user:
https://github.com/gazebosim/gz-sensors/blob/44c67b08bb4b5c44cff91501de9a23d659af985c/src/CameraSensor.cc#L864-L876

Can we do the same here?

Reply:
So you are recommending not exposing the method in CameraSensor just handle it in DepthCameraSensor the same way as? Current DepthCameraSensor calls PopulateInfo in the base class (CameraSensor) which does this check and assigns the correct frame to opticalFrameId which is returned by the added method.

Reply:
Ah, I didn't realize that. Never mind then.